app/web/modules/discrepancies/discrepancies.http.js

```diff
@@ -0,0 +1,33 @@
+const logger = require('./../../../lib/logger')
+
+const { Connector } = require('./../../../lib/pay-request')
+const { wrapAsyncErrorHandlers } = require('./../../../lib/routes')
+
+const search = async function searchTransaction (req, res) {
+  res.render('discrepancies/report')
+}
+
+const getDiscrepancyReport = async function searchTransaction (req, res) {
+  const chargeIds = req.body.search_string.split(',')
```

Comment:
If nonsense values are entered here will the handling of Connectors response be a clear enough user experience? I'm guessing the answer is yes but if not we could do a simple check to make sure the IDs passed in seem reasonable.

Reply:
connector will 404. Kind of fine for now